packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -18,7 +18,7 @@ import {
 } from "@fluidframework/runtime-definitions";
 import { generateErrorWithStack, TelemetryDataTag } from "@fluidframework/telemetry-utils";
 
-interface IResponseException extends Error {
+export interface IResponseException extends Error {
```

Comment:
is this export still needed? i don't see it being used. I see it imported to containerRuntime, but doesn't appear to be used after that.

Reply:
It is used in the `getAliasedDataStore` method of ContainerRuntime to check if `getDataStore` failed because of a 404 error.

Reply:
i don't think it's worth it to export. the above typing also doesn't really make sense as there is no type saftey due to the usage of `as`. the above code will literally look at any code value on any error.

Reply:
Removed the export.